raiden_contracts/tests/fixtures/base/utils.py

```diff
@@ -124,6 +133,9 @@ def get(txn_hash, message=None, additional_gas=0):
         print('----------------------------------')
         print('GAS USED ' + message, gas_used + additional_gas)
         print('----------------------------------')
+        gas_measurement_results[message] = gas_used + additional_gas
+        with contracts_gas_path().open(mode='w') as target_file:
+            target_file.write(json.dumps(gas_measurement_results))
```

Comment:
Adding `sort_keys=True, indent=4` to `json.dumps` will give more readable output with cleaner diffs.

Reply:
Very good idea.  Will do.

Reply:
Done.  Now just waiting for CI.